scripts/mkcloud

```diff
@@ -206,7 +206,7 @@ function show_environment
     echo "-------------------------------"
     env | grep -i "^want_"
     echo "-------------------------------"
-    column -s"," -t $timing_file_host | tee ${timing_file_host}.txt
+    sort -t, -k3,3 -k1,1 $timing_file_host | column -s"," -t  | tee ${timing_file_host}.txt
```

Comment:
Do you really want to mix it up like this?
This changes the absolut order of when happened what. I think that this will make it harder to find correlations.
Why not have the mixed file (sorted by type) additionally?

Reply:
I think we can live with this. 
Relative order of steps and proposal is not really important. 
But order is important same type